src/pocketmine/entity/Entity.php

```diff
@@ -1383,10 +1383,20 @@ protected function switchLevel(Level $targetLevel){
 		return true;
 	}
 
+	/**
+	 * @deprecated Use asPosition() instead getPosition()
+	 * 
+	 * @return Position
+	 */
 	public function getPosition(){
 		return new Position($this->x, $this->y, $this->z, $this->level);
 	}
 
+	/**
+	 * @deprecated Use asLocation() instead getLocation()
```

Comment:
I don't think there's any need for deprecating these. While entities are indeed positions, it also does make sense to be able to say Entity->getPosition() or Entity->getLocation(). The new methods are intended to make life easier for developers, yes?